### ISSUE TITLE
add authors credit!

### ISSUE THREAD

**Issue by santolucito** [Barnard-PL-Labs/tsltools#18]
https://github.com/Barnard-PL-Labs/tsltools/blob/ef94cb9237fdbd0bede08caac6b6a72ea3602629/docs/tsl_intro_blogpost.md?plain=1#L13

@Rheakk @nupurd89 @dc3347

**Comment by dc3347** [comment]
Update 3/22/22: I worked with Rhea to add markdown to html after much time trying to figure out how to add js to md file but not finding a solution. We were able to add the md file directly to html, but had to add the button beneath the entire md file (see Rhea's post for commit numbers). Also, it only shows up in vscode, not on github pages for some reason (see screenshot below). We also added author’s credit to the markdown file and added back the tonejs directory.
<img width="1338" alt="Screen Shot 2022-03-21 at 10 14 37 PM" src="https://user-images.githubusercontent.com/55637930/159394472-a1f1ada2-0352-4d51-a9a3-24cf34e86bcd.png">

**Comment by Rheakk** [comment]
Update: Danielle and I added author's credit ([ce676f1](https://github.com/Barnard-PL-Labs/tsltools/commit/ce676f17fdc4873518503e04aef7938467229c6f)), we also spent some time on Monday trying to add javascript to a markdown, but we still couldn't find a way to do this well. We were able to preserve the markdown formatting in an html file, and add buttons underneath each example - however, only the last button plays music ([a122e44](https://github.com/Barnard-PL-Labs/tsltools/commit/a122e44161bc327c9194e8c6f8b6b484a7890610)). We also added back the tonejs directory with the music2.js file ([d674373](https://github.com/Barnard-PL-Labs/tsltools/commit/d674373aca3d29d1a4bc7329f29b82d032a73385)).
